src/common/components/dataTable.js

```diff
@@ -0,0 +1,72 @@
+import React, { Fragment } from "react";
+import moment from "moment/moment";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableRow from "@material-ui/core/TableRow";
+import { makeStyles } from "@material-ui/core/styles";
+import ErrorIcon from "@material-ui/icons/Error";
+
+const useStyles = makeStyles(theme => ({
+  listItem: {
+    paddingBottom: "0px",
+    paddingTop: "0px"
+  },
+  table: {
+    border: "1px solid rgba(224, 224, 224, 1)"
+  },
+  abnormalColor: {
+    color: "#ff4f33"
+  }
+}));
+
+const dataTable = ({ tableDataToShow }) => {
+  const classes = useStyles();
+  return (
+    <div>
+      {tableDataToShow
```

Comment:
instead of passing an object as a prop just pass list of observations directly as a prop. And since it is a list of observations it makes sense to call the prop also observations.